docker/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -x
+#########Change ME###############
+export PYVER=2.7
+export PYV=27
+export CUDA_VERSION=10
+export NVIDIA_BUILD_ID=12345
+export DEPS_IMAGE=dali_cu${CUDA_VERSION}.deps
+export BUILDER=dali_${PYV}_cu${CUDA_VERSION}.build
+export RUN_IMG=dali_${PYV}_cu${CUDA_VERSION}.run
+#################################
+pushd ../
+docker build -t ${DEPS_IMAGE} --build-arg "USE_CUDA_VERSION=${CUDA_VERSION}" -f Dockerfile.deps .
+echo "Build image:" ${BUILDER}
+test $? -eq 0 && docker build -t ${BUILDER} --build-arg "DEPS_IMAGE_NAME=${DEPS_IMAGE}" --build-arg "PYVER=${PYVER}" --build-arg "PYV=${PYV}" --build-arg "NVIDIA_BUILD_ID=${NVIDIA_BUILD_ID}" .
```

Comment:
Are those ` test $? -eq 0 ` checks necessary? They will check if `echo ` was ok, not the previous docker command or anything important.

Reply:
Done